modules/ROOT/pages/message-enricher.adoc

```diff
@@ -89,3 +89,50 @@ You can find information on using a variable with any enricher listed on these p
 * xref:non-mel-expressions-configuration-reference.adoc[Default enrichers] are loaded at runtime
 * Schema documentation for the http://www.mulesoft.org/docs/site/current/schemadocs/schemas/mule_xsd/elements/enricher.html[enricher element]
 * API documents for https://www.mulesoft.org/docs/site/3.8.0/apidocs/org/mule/enricher/MessageEnricher.html[org.mule.core.enricher.MessageEnricher]
+
+== Mule XML Example
+
+The Mule flow in the following example shows how to use a Message Enricher (<enricher/>) to modify content of a Mule variable. It uses Set Variable to set the initial value of the Mule variable called `greeting`. It also shows how to use Set Variable and Transform Message to enrich the value of `greeting` and how to pass the value of the Mule variable to the payload. In addition, it demonstrates differences between the scope of a payload and variable that are defined or modified within the scope of the Processor Chain.
```

Comment:
Should (<enricher/>) have any formatting, perhaps monospace?

Reply:
good catch, fixing